### PR TITLE
Mvc cleanup

### PR DESCRIPTION
**Pull request by marcincichon**
@RobertWDLowe @ollietreend can you review?

### CODE REVIEW

File: Loader.php [hunk 1]
```diff
@@ -3,37 +3,41 @@
...
-    $this->is_plugin = strpos($instance_dir, realpath(WP_PLUGIN_DIR)) === 0;
-    $this->views_dir = $this->is_plugin ? $instance_dir.'/'.MVC_VIEWS_DIR : MVC_VIEWS_PATH;
-    $this->models_dir = $this->is_plugin ? $instance_dir.'/'.MVC_MODELS_DIR : MVC_MODELS_PATH;
+    $this->settings = [
+      'views_dir' => 'views',
+      'models_dir' => 'models'
+    ];
+
+    $this->template_path = get_template_directory() . $this::DS;
+    $this->views_path = $this->template_path . $this->settings['views_dir'] . $this::DS;
```
Comment (RobertWDLowe) [inline Loader.php]: Does this need to be a variable? Could just be 'views' string.

**Comment by RobertWDLowe** [comment]
Looks good

File: dw-mvc.php [hunk 2]
```diff
@@ -11,100 +11,99 @@
...
-  add_rewrite_tag('%controller%', '([^&]+)');
-  add_rewrite_tag('%param_string%', '([^&]+)');
+  /** !!! TODO: to be removed as soon as we remove the issues in the doc revs plugin (coming soon)
+   */
+  function action_unhook_document_revisions_auth() {
+    global $wpdr;
+    if(isset($wpdr)) {
+      remove_action('after_setup_theme', [&$wpdr, 'auth_webdav_requests']);
+    }
+  }
```
Comment (ollietreend) [inline dw-mvc.php]: @marcincichon This can be removed as soon as @RobertWDLowe has deployed `dw-document-revisions`.

**Comment by ollietreend** [comment]
@marcincichon I've added one comment – but apart from that it looks good.

**Pull request merged**